scripts/setup-release-on-ubuntu.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+RELEASE_VERSION=${RELEASE_VERSION:=0.24.0}
+RELEASE_FILE="https://github.com/aeternity/epoch/releases/download/v${RELEASE_VERSION}/epoch-${RELEASE_VERSION}-ubuntu-x86_64.tar.gz"
+TEMP_RELEASE_FILE=${TEMP_RELEASE_FILE:=/tmp/epoch.tgz}
+TARGET_DIR=${TARGET_DIR:=$HOME/epoch}
+EPOCH_CONFIG=${EPOCH_CONFIG:=$TARGET_DIR/epoch.yaml}
+NPROC=$(nproc)
+
+echo ""
+read -sp "Please provide a secret key passphrase to be used for your keys: " keypassphrase
+echo ""
+
+echo -e "\nPrepare host system and install dependencies...\n"
+sudo apt-get update
+sudo apt-get upgrade -y
+sudo apt-get install curl jq tmux htop libsodium23
+
+echo -e "\nInstall release...\n"
+curl -L "${RELEASE_FILE}" > "${TEMP_RELEASE_FILE}"
+rm -rf "${TARGET_DIR}"
+mkdir -p "${TARGET_DIR}"
+tar -C "${TARGET_DIR}" -xzf "${TEMP_RELEASE_FILE}"
+
+echo -e "\nConfigure release...\n"
+cd "${TARGET_DIR}"
+BENEFICIARY=$(./bin/epoch keys_gen ${keypassphrase} | awk -F ': ' '{ print $2; }')
+
+cat <<EOF > "${EPOCH_CONFIG}"
+sync:
+    port: 3115
+    external_port: 3015
+keys:
+    dir: keys
+    password: "${keypassphrase}"
+http:
+    external:
+        port: 3013
+    internal:
+        port: 3113
+websocket:
+    channel:
+        port: 3014
+    internal:
+        port: 3114
+mining:
+    beneficiary: "${BENEFICIARY}"
+    autostart: true
+    cuckoo:
+        miner:
+            executable: lean30
```

Comment:
is there a reason this miner is pre-configured, not a `mean` variation

Reply:
Mainly because I didn't get `mean` to work on Windows. However, my sample size is 1.

Reply:
`mean30s-generic` and `mean30s-avx2` both seem to work for me, but as well just sample size 1

Reply:
Does `lean30` work for you? Then this would be the known working configuration on at least 2 systems.

Reply:
Yes, it does

Reply:
Then I'd propose to keep `lean30`. The whole miner configuration needs better documentation anyway, but that is a different topic.

Reply:
The default miner we distribute with other packages and in the docs is `mean30`. `lean30` is kind of useless and for testing purposes only.

@tolbrino Can you provide further details about your issues with `mean30` so we can work it out and change this to it.

Reply:
@dincho I ran into the issue described in the [wiki](https://github.com/aeternity/epoch/wiki/Troubleshooting) (first entry). However, I can't reproduce it now and `mean30s-generic` works, so I guess I can remove the cuckoo configuration and simply use the default there.

Reply:
@tolbrino That would be great